tests/runtime/config

```diff
@@ -0,0 +1,25 @@
+NAME config as env var
```

Comment:
not env var, right?
```suggestion
NAME config
```

Reply:
Maybe the name is confusing but I want to test that the env variable style e.g. `BPFTRACE_` works for setting the config (as opposed to the test below)

Reply:
Ah, right, I just misunderstood it. I think that the current name is ok.